data/auxiliary/gather/ldap_query/ldap_queries_default.yaml

```diff
@@ -10,19 +10,42 @@ queries:
       - cACertificateDN
       - dNSHostname
       - certificateTemplates
-  - action: ENUM_ADCS_CERT_TEMPLATES
-    description: 'Enumerate ADCS certificate templates.'
+      - objectGUID
+      - caCertificate
+    referemces:
+      - https://aaroneg.com/post/2018-05-15-enterprise-ca/
+  - action: ENUM_ADCS_ESC1_VULN_TEMPLATES
+    description: 'Enumerate ADCS ESC1 vulnerable certificate templates.'
     base_dn_prefix: 'CN=Certificate Templates,CN=Public Key Services,CN=Services,CN=Configuration'
-    filter: '(objectClass=pkicertificatetemplate)'
+    filter: '(&(objectclass=pkicertificatetemplate)(!(mspki-enrollment-flag:1.2.840.113556.1.4.804:=2))(|(mspki-ra-signature=0)(!(mspki-ra-signature=*)))(|(pkiextendedkeyusage=1.3.6.1.4.1.311.20.2.2)(pkiextendedkeyusage=1.3.6.1.5.5.7.3.2) (pkiextendedkeyusage=1.3.6.1.5.2.3.4))(mspki-certificate-name-flag:1.2.840.113556.1.4.804:=1))'
```

Comment:
Would be good to redo this and create another filter just on this object class to gather the list of Enterprise CAs. I didn't fully understand this field until I read the Certified Preowned paper a bit more and now realizing I was a bit incorrect in my theory here 👍